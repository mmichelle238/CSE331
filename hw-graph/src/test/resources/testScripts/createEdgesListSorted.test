# create a graph and 3 nodes and create edges
# between them. This test ensures that the list
# of the children of a node is sorted alphabetically

CreateGraph A
AddNode A n1
AddNode A n2
AddNode A n3
AddEdge A n3 n2 e31
AddEdge A n3 n1 e32
AddEdge A n3 n3 e33
ListChildren A n3
ListNodes A