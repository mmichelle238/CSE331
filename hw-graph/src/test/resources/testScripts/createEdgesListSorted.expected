# create a graph and 3 nodes and create edges
# between them. This test ensures that the list
# of the children of a node is sorted alphabetically

created graph A
added node n1 to A
added node n2 to A
added node n3 to A
added edge e31 from n3 to n2 in A
added edge e32 from n3 to n1 in A
added edge e33 from n3 to n3 in A
the children of n3 in A are: n1(e32) n2(e31) n3(e33)
A contains: n1 n2 n3